config/routes.rb

```diff
@@ -20,4 +22,7 @@
 
   # Add new route for "My Reservations"
   get '/my_reservations', to: 'reservations#my_reservations'
+
+  #  new route for creating a reservation
+  post '/reservations', to: 'reservations#create'

```

Comment:
Since you move the reservation routes into the namespace, we don't need those routes anymore.

Don't forget to move the reservation_controller file into the API/v1 folder too.